superset-frontend/src/explore/components/controls/FixedOrMetricControl.jsx

```diff
@@ -107,67 +100,88 @@ export default class FixedOrMetricControl extends React.Component {
     return (
       <div>
         <ControlHeader {...this.props} />
-        <Label onClick={this.toggle}>
-          {this.state.type === controlTypes.fixed && (
-            <span>{this.state.fixedValue}</span>
-          )}
-          {this.state.type === controlTypes.metric && (
-            <span>
-              <span style={{ fontWeight: 'normal' }}>metric: </span>
-              <strong>
-                {this.state.metricValue ? this.state.metricValue.label : null}
-              </strong>
-            </span>
-          )}
-        </Label>
-        <Panel
-          className="panel-spreaded"
-          collapsible
-          expanded={this.state.expanded}
-          onToggle={this.toggle}
+        <Collapse
+          ghost
+          css={css`
+            &.ant-collapse
+              > .ant-collapse-item.ant-collapse-no-arrow
+              > .ant-collapse-header {
+              border: 0px;
+              padding: 0px 0px 8px 0px;
+              display: inline-block;
+            }
+            &.ant-collapse-ghost
+              > .ant-collapse-item
+              > .ant-collapse-content
+              > .ant-collapse-content-box {
+              padding: 0px;
+
+              & .well {
+                margin-bottom: 0px;
+                padding: 8px;
+              }
+            }
+          `}
         >
-          <Panel.Collapse>
-            <Panel.Body>
-              <div className="well">
-                <PopoverSection
-                  title="Fixed"
-                  isSelected={type === controlTypes.fixed}
-                  onSelect={() => {
+          <Collapse.Panel
+            showArrow={false}
+            header={
+              <Label onClick={() => undefined}>
+                {this.state.type === controlTypes.fixed && (
+                  <span>{this.state.fixedValue}</span>
+                )}
+                {this.state.type === controlTypes.metric && (
+                  <span>
+                    <span style={{ fontWeight: 'normal' }}>metric: </span>
+                    <strong>
+                      {this.state.metricValue
+                        ? this.state.metricValue.label
+                        : null}
+                    </strong>
+                  </span>
+                )}
+              </Label>
+            }
+          >
+            <div className="well">
+              <PopoverSection
+                title="Fixed"
```

Comment:
Another out-of-scope item since it's not pertinent to the intent of this PR, but maybe worth adding a `t()` here and the "Based on a metric" title below?

Reply:
Done.